applications/SLF80037.COM.asm

```diff
@@ -0,0 +1,911 @@
+                    DB $02,$00,$00,$22,$48,$42,$02,$00,$00,$02,$00,$00,$02,$00
+                    DB $00,$02,$00,$00,$02,$00,$00,$02,$00,$00,$02,$00,$00,$08
+                    DB $48,$69,$08,$6F,$48,$0E,$48,$00,$02,$00,$00,$02,$00,$00
+                    DB $02,$00,$00,$02,$00,$00,$02,$00,$00,$22,$48,$48,$08,$48
+                    DB $6F,$12,$48,$00,$02,$00,$00,$02,$00,$00,$02,$00,$00,$02
+                    DB $00,$00,$02,$00,$00,$02,$00,$00,$02,$00,$00,$02,$00,$00
+                    DB $0F,$6B,$00,$13,$6B,$00,$08,$6D,$68,$02,$00,$00,$02,$00
+                    DB $00,$22,$48,$46,$02,$00,$00,$02,$00,$00,$02,$00,$00,$02
+                    DB $00,$00,$02,$00,$00,$02,$00,$00,$AA,$20,$60,$A6,$60,$20
+                    DB $2E,$44,$40,$08,$6F,$40,$8E,$00,$00,$92,$00,$00,$D2,$30
+                    DB $00,$0A,$39,$27,$AA,$21,$60,$A6,$60,$21,$26,$44,$40,$08
+                    DB $40,$6F,$02,$00,$00,$96,$00,$00,$02,$00,$00,$0A,$3A,$27
+                    DB $AA,$22,$60,$A6,$60,$22,$2E,$44,$42,$08,$6F,$42,$02,$00
+                    DB $00,$02,$00,$00,$D2,$31,$00,$0A,$27,$39,$AA,$23,$60,$A6
+                    DB $60,$23,$26,$44,$42,$08,$42,$6F,$02,$00,$00,$02,$00,$00
+                    DB $D2,$32,$00,$0A,$27,$3A,$AA,$24,$60,$A6,$60,$24,$2E,$44
+                    DB $44,$02,$00,$00,$02,$00,$00,$02,$00,$00,$02,$00,$00,$9A
+                    DB $00,$00,$AA,$25,$60,$A6,$60,$25,$26,$44,$44,$02,$00,$00
+                    DB $02,$00,$00,$02,$00,$00,$02,$00,$00,$9E,$00,$00,$AA,$26
+                    DB $60,$A6,$60,$26,$2E,$44,$46,$08,$6F,$46,$02,$00,$00,$02
+                    DB $00,$00,$02,$00,$00,$02,$00,$00,$AA,$27,$60,$A6,$60,$27
+                    DB $26,$44,$46,$08,$46,$6F,$02,$00,$00,$02,$00,$00,$02,$00
+                    DB $00,$02,$00,$00,$02,$00,$00,$02,$00,$00,$02,$00,$00,$02
+                    DB $00,$00,$02,$00,$00,$02,$00,$00,$02,$00,$00,$02,$00,$00
+                    DB $02,$00,$00,$02,$00,$00,$02,$00,$00,$02,$00,$00,$02,$00
+                    DB $00,$02,$00,$00,$02,$00,$00,$02,$00,$00,$02,$00,$00,$02
+                    DB $00,$00,$02,$00,$00,$02,$00,$00,$02,$00,$00,$02,$00,$00
+                    DB $02,$00,$00,$02,$00,$00,$02,$00,$00,$02,$00,$00,$02,$00
+                    DB $00,$02,$00,$00,$02,$00,$00,$02,$00,$00,$02,$00,$00,$02
+                    DB $00,$00,$E0,$00,$00,$E1,$00,$00,$E2,$00,$00,$E3,$00,$00
+                    DB $02,$00,$00,$02,$00,$00,$02,$00,$00,$02,$00,$00,$E4,$00
+                    DB $00,$E5,$00,$00,$E6,$00,$C3,$00,$00,$C3,$00,$00,$C3,$00
+                    DB $00,$C3,$00,$00,$C3,$00,$00,$C3,$00,$00,$C3,$00,$00,$C3
+                    DB $00,$00,$C3,$00,$00,$C3,$00,$00,$C3,$00,$00
+
+                    ; This strange string is inside this unreachable blob
+                    ; kept here, with its address, just for reference
+                    ; strange_str:                          ;[0768]
```

Comment:
These look like the strings in the DDT for disassembled opcodes. Is this a core dump of the original executable?

Reply:
Yes. That string note can be removed, since it is dirty memory like all the other unreachable code